p2p/net/mock/mock_test.go

```diff
@@ -460,7 +462,7 @@ func TestAdding(t *testing.T) {
 	if h1 == nil {
 		t.Fatalf("no network for %s", p1)
 	}
-
+	
```

Comment:
you'll want to make sure to run `go fmt` -- http://tip.golang.org/misc/git/pre-commit